src/mame/seibu/raiden.cpp

```diff
@@ -334,25 +348,25 @@ WRITE_LINE_MEMBER(raiden_state::vblank_irq)
 
 void raiden_state::raiden(machine_config &config)
 {
-	/* basic machine hardware */
-	V30(config, m_maincpu, XTAL(20'000'000)/2); /* NEC V30 CPU, 20MHz verified on pcb */
+	// basic machine hardware
+	V30(config, m_maincpu, XTAL(20'000'000)/2); // NEC V30 CPU, 20MHz verified on pcb
 	m_maincpu->set_addrmap(AS_PROGRAM, &raiden_state::main_map);
 
-	V30(config, m_subcpu, XTAL(20'000'000)/2); /* NEC V30 CPU, 20MHz verified on pcb */
+	V30(config, m_subcpu, XTAL(20'000'000)/2); // NEC V30 CPU, 20MHz verified on pcb
 	m_subcpu->set_addrmap(AS_PROGRAM, &raiden_state::sub_map);
 
-	z80_device &audiocpu(Z80(config, "audiocpu", XTAL(14'318'181)/4)); /* verified on pcb */
+	z80_device &audiocpu(Z80(config, "audiocpu", XTAL(14'318'181)/4)); // verified on pcb
 	audiocpu.set_addrmap(AS_PROGRAM, &raiden_state::seibu_sound_map);
 	audiocpu.set_irq_acknowledge_callback("seibu_sound", FUNC(seibu_sound_device::im0_vector_cb));
 
 	config.set_maximum_quantum(attotime::from_hz(12000));
 
-	/* video hardware */
+	// video hardware
 	BUFFERED_SPRITERAM16(config, m_spriteram);
 
 	screen_device &screen(SCREEN(config, "screen", SCREEN_TYPE_RASTER));
 	screen.set_refresh_hz(59.60); // verified on pcb */
-	screen.set_vblank_time(ATTOSECONDS_IN_USEC(2500) /* not accurate */);
+	screen.set_vblank_time(ATTOSECONDS_IN_USEC(2500) // not accurate);
```

Comment:
You've commented out the closing parenthesis and semicolon. What was the comment doing *inside* the argument list anyway? I suspect a vast network of saboteurs conspiring to undermine the Seibu drivers. This message will self-destruct in...5...4...

Reply:
That is really weird. Fixed it anyway